lib/index.js

```diff
@@ -44,7 +44,7 @@ Zen.indexHtml = function indexHtml (pageType, forS3) {
     deps.push('node_modules/svelte/store.umd.js', 'node_modules/fuzzysort/fuzzysort.js', 'svelte/mini.js', 'svelte/command.js')
   }
   deps.push(`lib/${pageType}.js`) // after Zen dependencies, but before user code
-  let entries = (Zen.webpack && Zen.webpack.compile && Zen.webpack.compile.entrypoints) || []
+  let entries = (Zen.webpack && Zen.webpack.entrypoints) || []
```

Comment:
No changes needed... 

@rfauver each time I think about this zen stuff, I grow more and more interested in splitting webpack off entirely... 

I wonder if we can: 
1. change the zen config to receive `entries`, not the entire webpack config. 
2. expose most of `runIfCodeChanged` via `sendStatus` as a method on Zen. 


Allowing for a simpler, and programatic zen api like this: 
```
# zen.config.js
import Zen from 'zen';
import webpack from 'webpack';
import webpackConfig from './webpack.config.js';

// plain old webpack. no zen modifications or anything
const compiler = webpack(webpackConfg);

// 1: when that webpack finishes, start a zen instance from its entries
compiler.hooks.done.tap('Zen', (stats, callback) => {
      const entrypoints = stats.compilation.entrypoints.get('bundle').chunks.map(chunk => chunk.files[0])
      testServer = new Zen({
          entries: webpack.entrypoints
      });
})
// 2: optional: 
// if that webpack updates, also update the zen instance
const compiler.watch({}, (err, stats) => {
  if (testServer && stats.done) {
      testServer.update();
  }
});
```